### PR TITLE
Preprod: movie: The Day After Tomorrow (2004) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Think global warming is going to transform the earth into a hot house, rather than a glaciated abyss? Think again.

### CODE REVIEW

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 1]
```diff
@@ -4449,6 +4449,46 @@ high treason [Lieutenant Dill made a failed attempt to place the bug compromised
...
+2004-05-17
+
+:: References
+https://en.wikipedia.org/wiki/The_Day_After_Tomorrow
+
+:: Ratings
+2 <paul>
+
+:: Choice Themes
+ice age [The film is principally about Earth suddenly entering into a new ice age.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:4476]: Should probably rename this to something like "new ice age" to make clear it's not a (pre)history theme

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 2]
```diff
@@ -4449,6 +4449,46 @@ high treason [Lieutenant Dill made a failed attempt to place the bug compromised
...
+
+:: Major Themes
+human vs. nature [The people of the Northern Hemisphere were brought to the brink by a trio of enormous snow hurricanes.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:4479]: to the brink of what? Yeah I know, but imho better to complete such expressions for clarity.

**Review by odinlake — APPROVED** [review]

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt]: Renamed this one of **global cooling**.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt]: Reworked this motivation to describe the protagonists' experience.

**Pull request merged**